book_app/authors/tests.py

```diff
@@ -1,3 +1,115 @@
+from datetime import date
+
 from django.test import TestCase
+from django.urls import reverse
+from rest_framework.test import APIClient
+from rest_framework import status
+
+from authors.serializers import AuthorSerializer
+from authors.models import Author
+
+
+AUTHORS_URL = reverse('author:author-list')
+
+def detail_url(author_id):
+    return reverse('author:author-detail', args=[author_id])
+def create_author(**params):
+    defaults = {
+        'first_name': 'Ken',
+        'last_name': 'Follett',
+        'date_of_birth': date(1949, 6, 5),
+        'description': 'British author of thrillers and historical novels'
+    }
+    defaults.update(params)
+    author = Author.objects.create(**defaults)
+    return author
```

Comment:
You can include those method in the test class. You can make it private. Read a bit about factoryboy for creating objects in tests. It comes in handy :)